web/projects/ui/src/app/routes/portal/routes/marketplace/components/controls.component.ts

```diff
@@ -36,7 +36,7 @@ import { ApiService } from 'src/app/services/api/embassy-api.service'
   template: `
     @if (localPkg) {
       @if (
-        localPkg.stateInfo.state === 'installed' && (localPkg | toManifest);
+        localPkg.stateInfo.state && (localPkg | toManifest);
```

Comment:
Youre just checking if state exists?

Reply:
yes since `state: "installing" | "restoring" | "installed" | "removing" | "updating"`, and its needed to complete the `localManifest` type

Reply:
but it's always there, you're just testing it for truthiness here, and it's always truthy...